### PR TITLE
tidy up (non-ticket changes from ECDC-3447)

### PR DESCRIPTION
**Pull request by ggoneiESS**
## Issue
Tidying the branch in order to make it clear what is what

## Description of work
ECDC-3447 branch currently contains changes that are not related to the ticket but which consist of useful changes to clarify code or comments
## Checklist

- [ N/A ] Changes have been documented in `changes.md`
- [ N/A ] Documentation has been updated in `documentation` directory

### CODE REVIEW

File: src/WriterRegistrar.h [hunk 1]
```diff
@@ -69,9 +69,10 @@ void clear();
 template <typename Module> class Registrar {
 public:
   /// \brief Register the writer module given in template parameter `Module`
-  /// under the
-  /// identifier `FlatbufferID`.
-  ///
+  /// under the identifier `FlatbufferID`.
+  /// ecdc-3447 this comment looks out of date, correct call is:
```
Comment (mattclarke) [inline src/WriterRegistrar.h:73]: Let's have a quick chat about this one.
I need a bit more context

**Review by mattclarke — COMMENTED** [review]
Nice job!
Let's just have a quick chat then we can merge it.

File: src/WriterRegistrar.h (hunk 1 above)
Comment (ggoneiESS) [reply src/WriterRegistrar.h]: Looking back at the other branch it seems I made this comment early on in the process.  Looking now in context of more knowledge about the codebase and how the modules are registered this comment isn't valid, but I've expanded it to make the link with the other important parts of the stream registration more explicit.

**Review by mattclarke — APPROVED** [review]

**Comment by mattclarke** [comment]
Force merging as these changes are cosmetic and unrelated to the flakiness of the integration tests.

**Pull request merged**